examples/basic-documentations/basic_documentations.out

```diff
@@ -1,4 +1,4 @@
 # To run this sample, navigate to the directory that contains the
 # `.bal` file, and execute the `ballerina run` command.
-$ ballerina run documentation.bal
+$ ballerina run basic_documentations.bal
```

Comment:
Instead of saying `basic_documentations`, shall we rename it to `basic_documentation`?

Reply:
Done